packages/admin/src/pages/scheme/[schemeId]/[exportId]/index.page.tsx

```diff
@@ -0,0 +1,232 @@
+  )
+    return generateErrorPageRedirect(
+      'There are no submissions available to view or download.',
+      '/dashboard'
+    );
+
+  return {
+    props: {
+      formAction: process.env.SUB_PATH + resolvedUrl,
+      csrfToken: res.getHeader('x-csrf-token') as string,
+      individualApplicationsHref: `/scheme/${schemeId}/${exportId}/download-individual`,
+      superZipLocation: superZipLocation,
+      schemeName: grantScheme.name,
+      availableSubmissionsTotalCount: availableSubmissionsTotalCount,
+      unavailableSubmissionsTotalCount: unavailableSubmissionsTotalCount,
+      unavailableSubmissions: unavailableSubmissions,
+      schemeId: schemeId,
+      exportId: exportId,
+    },
+  };
+};
+
+export const CompletedSubmissions = ({
+  formAction,
+  csrfToken,
+  schemeName,
+  individualApplicationsHref,
+  availableSubmissionsTotalCount,
+  unavailableSubmissionsTotalCount,
+  unavailableSubmissions,
+  superZipLocation,
+  schemeId,
+  exportId,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const tableHeadColumns = [
+    {
+      name: <>{true && 'Organisation'}</>,
+      width: 'three-quarters',
+    },
+    {
+      name: 'Action',
+      isVisuallyHidden: true,
+    },
+  ] as TheadColumn[];
+
+  const tableRows = unavailableSubmissions.map((submission) => {
+    return {
+      cells: [
+        {
+          content: <>{true && submission.name}</>,
+        },
+        {
+          content: (
+            <div className="govuk-!-text-align-right">
+              <CustomLink
+                href={`/scheme/${schemeId}/${exportId}/view/${submission.submissionId}`}
+                ariaLabel={`View submission "${submission.name}"`}
+              >
+                View
+              </CustomLink>
+            </div>
+          ),
+        },
+      ],
+    };
+  });
+
+  return (
+    <>
+      <Meta title={`Download applications - Manage a grant`} />
+      <div className="govuk-grid-row govuk-!-padding-top-7 govuk-!-margin-0 govuk-!-margin-bottom-6">
+        <div className="govuk-grid-column-full-width">
+          <div className="govuk-!-width-two-thirds">
+            {unavailableSubmissionsTotalCount > 0 && (
+              <DownloadMessage count={unavailableSubmissionsTotalCount} />
+            )}
+
+            <h1 className="govuk-heading-l">{schemeName}</h1>
+
+            <h2 className="govuk-heading-m">Applications submitted</h2>
+
+            <p className="govuk-body">
+              Your grant has{' '}
+              <b>
+                {availableSubmissionsTotalCount}{' '}
+                {availableSubmissionsTotalCount === 1
+                  ? 'application'
+                  : 'applications'}
+              </b>{' '}
+              available to download.
+            </p>
+          </div>
+          <FlexibleQuestionPageLayout
+            fieldErrors={[]}
+            formAction={formAction}
+            csrfToken={csrfToken}
+          >
```

Comment:
Is this component needed? I might be missing it but it doesn't seem to be doing anything

Reply:
Will be in next push!